app/controllers/editions_controller.rb

```diff
@@ -27,6 +30,23 @@ def show
   alias_method :metadata, :show
   alias_method :unpublish, :show
 
+  def update
+    @resource.assign_attributes(permitted_params)
+
+    if @resource.update!
```

Comment:
I'm not sure what `@resource.update!` is actually doing/comes from—if `@resource` is the model object, shouldn't we be calling `save`/`save!`? I think the `InheritedResources` base class maybe causing confusion, and wonder whether we should stop using it.

It seems like it is throwing an exception when there is a validation error, as I'm seeing both the validation message and the error message from `render_show_page_with_error` when I try and save an Answer with a blank title, for example.

Reply:
@mtaylorgds I have updated this section of the controller to call `save` in a way that seems to work and doesn't show an error message with the validation method. It's not really addressing your concerns about the `InheritedResources` class though as I don't fully understand why that is being used here and the consequences of doing it differently. I'm feeling that may be a slightly separate piece of work if we can get this to work as expected for the present.